var/spack/repos/builtin/packages/py-h5py/h5py370_hdf51140.patch

```diff
@@ -0,0 +1,146 @@
+--- a/h5py/api_types_hdf5.pxd	2023-03-30 22:22:49
++++ b/h5py/api_types_hdf5.pxd	2023-03-30 22:25:19
+@@ -235,40 +235,100 @@
+     H5FD_MPIO_INDEPENDENT = 0,
+     H5FD_MPIO_COLLECTIVE
+ 
++  # File driver identifier type and values
```

Comment:
I'm guessing these code changes in this patch were determined by comparing h5py 3.7.0 and 3.8.0. How well are they tested? For example, did the recent container work help verify that these changes are okay for our needs?

Reply:
@climbfuji thanks for the reference to the PR in h5py. That is quite helpful!